patches/react-beautiful-dnd+13.1.1.patch

```diff
@@ -0,0 +1,259 @@
+diff --git a/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.cjs.js b/node_modules/react-beautiful-dnd/dist/react-beautiful-dnd.cjs.js
```

Comment:
We tried to fix upstream rather than patch? Patching lots of codes seems not ideal